x/deployment/keeper/keeper_test.go

```diff
@@ -210,6 +223,41 @@ func Test_OnDeploymentClosed(t *testing.T) {
 	})
 }
 
+func Test_CloseGroup(t *testing.T) {
+	ctx, keeper := setupKeeper(t)
+	groups := createActiveDeployment(t, ctx, keeper)
+	gIdx0, gIdx1 := 0, 1
+
+	t.Run("close-group-0", func(t *testing.T) {
+		assert.NoError(t, keeper.OnCloseGroup(ctx, groups[gIdx0]))
+	})
+	t.Run("group-1-open", func(t *testing.T) {
+		s := groups[gIdx1]
+		t.Logf("%v", s)
```

Comment:
?

Please just follow the pattern from [`Test_OnLeaseClosed`](https://github.com/ovrclk/akash/blob/f42f3233705f09750195a04a659d97c2ec7e55a2/x/deployment/keeper/keeper_test.go#L173-L191) for this entire test function.